frontend/css/layout.css

```diff
@@ -14,21 +16,18 @@ a {
 
 .account {
     justify-content: flex-end;
-    padding-right: 20px;
     border-bottom: 0.5px solid lightgray;
     font-size: 14px;
 }
 
 .account-links {
     display: flex;
     list-style: none;
-    gap: 20px;
 }
 
 /* Search section */
 
 .menu {
```

Comment:
What do you think about specifying the width of the children within the scope of the `menu` class? If we take inspiration from the BEM naming standard we could do something like 

```
.menu {
    border-bottom: 2px solid lightgray;

    .menu__item {
        width: 33%;
        // or "flex: 1;", although in that case we need to make sure that the each child item has the right width which pushes the responsibility down from this pure layout class
        overflow: hidden;
    }

    .menu__item + .menu__item { // add space between the menu items
        margin-left: var(--spacing-small);
    }
}
```

And then each of the classes for child items (`logo`, and`search-container`) wouldn't need to specify `flex: 1`. This way we can keep the layout rules more encapsulated rather than spread out on the more low level classes.